website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/troubleshooting-tips.mdx

```diff
@@ -0,0 +1,104 @@
+Scroll down to **Related Links > Flow Context > Open Context Record** and find the **Flow engine log entries** tab. 
+
+### Application logs
+If workflow or flow logs aren't sufficient, navigate to **All > System Log > Application Logs.**
+Filter the records by setting the **Application** to "Terraform" for better visibility. 
+Search for logs around the time your issue occurred and you can often find HTTP status codes and detailed error messages.
+
+### Outbound requests
+For stats on outgoing API calls from your ServiceNow instance to any external endpoint, including HCP Terraform, navigate  to **All > System Logs > Outbound HTTP Requests.** 
+Customize the table view by adding columns like "URL," "URL Path," and "Application scope." 
+Logs from the Catalog app are marked with the "x_325709_terraform" scope. 
+Use filters and grouping to analyze trends effectively.
+
+## Enable email notifications
+
+To receive updates on your requested item tickets, you can enable email notifications by:
+
+1. Logging into your ServiceNow instance as an administrator. 
+   **Click System Properties > Email Properties** to configure email notifications. 
+   In the **Outbound Email Configuration** panel, select **Yes** next to the check-box with the email to which you would like these requests to be sent.
+
+1. To ensure you have relevant notifications configured in your instance, Navigate to **System Notification > Email > Notifications.**
+   Search for "Request Opened" and "Request Item Commented" and ensure they are activated.
+
+1. Reaching out to ServiceNow customer support if you run into any issues with the global configurations.
+
+## Common problems
+
+This section details frequently encountered issues and how they can be resolved. 
+
+### Failure to create a workspace
+
+If you order the "create a workspace" catalog item and  nothing happens in ServiceNow and HCP Terraform does not create a workspace then there are several possible reasons why: 
+
+1. Bad Credentials such as an incorrect token, hostname or organization name.   
+   1. Make sure to use a **Team API Token**. This can be found in HCP Terraform under "API Tokens". 
+   1. Make sure the team API token has the correct permissions. 
+   1. Double-check your organization name by copying and pasting it from HCP Terraform or Terraform Enterprise.
+   1. Double-check your host name. 
+   1. Make sure you created your team API token  in the same organization you are using 
+   1. Test your configuration. First click **Update** to process any changes then **Test Config to make sure the connection is working. 
+1. Wrong VCS Configuration
+   1. **Identifier** field should not have any spaces. The format is: username/repo_name.
+   1. **Name** can be anything, but it is better to avoid special characters as per naming convention.
+   1. Double-check the OAuth token ID in your HCP Terraform/Terraform Enterprise settings. In your HCP Terraform, navigate to **Organization > Settings > Provider.**
```

Comment:
```suggestion
Ensure your HCP Terraform token, hostname, and organization name is correct.

1. Make sure to use a **Team API Token**. This can be found in HCP Terraform under "API Tokens". 
1. Ensure the team API token has the correct permissions. 
1. Double-check your organization name by copying and pasting it from HCP Terraform or Terraform Enterprise.
1. Double-check your host name. 
1. Make sure you created your team API token  in the same organization you are using 
1. Test your configuration. First click **Update** to process any changes then **Test Config to make sure the connection is working.
  
Verify your VCS configuration.

1. The **Identifier** field should not have any spaces.  The ServiceNow Service Catelog Integration requires that you format repository names in the `username/repo_name` format.
1. The **Name** can be anything, but it is better to avoid special characters as per naming convention.
1. Double-check the OAuth token ID in your HCP Terraform/Terraform Enterprise settings. To retrieve your OAuth token ID, navigate to your HCP Terraform organization's settings page, then click **Provider** in the left navigation bar under **Version Control**.
```